server/controllers/entities/lib/entities.coffee

```diff
@@ -97,6 +97,8 @@ module.exports = entities_ =
   putUpdate: (params)->
     { userId, currentDoc, updatedDoc } = params
     _.types [ userId, currentDoc, updatedDoc ], ['string', 'object', 'object']
+    # Checking that there is actually an update is let to the responsability
+    # of consumers: empty patches at this stage will throw 500 errors
```

Comment:
suggestion : 

    # It is to the consumers responsability to check if there is an update: 
    # empty patches at this stage will throw 500 errors